example/windows/datePickerTest/ReactPackageProvider.cpp

```diff
@@ -0,0 +1,20 @@
+#include "pch.h"
```

Comment:
I'm not sure. Looking at Kai's Picker PR (https://github.com/react-native-community/react-native-picker/pull/58/files#diff-04c6e90faac2675aa89e2176d2eec7d8) the license is not present everywhere. I can add it to the new files I'm creating.

Reply:
I'm not sure either but agree we should be consistent across all community modules :) you might want to check what CELA guidance is or if @cglein has it.

Reply:
Yeah I'm not sure either 😂
I think we should have it in all files, I'll update the template to include it.

Reply:
not in the template since that's what apps get - but in any code that we write ourselves (RNW core, viewmanagers, native modules, and community modules), makes sense to me

Reply:
@chrisglein should we put the license at the top of all new files?

Reply:
I've added the license to all .h/cpp/.idl files. WIll push with the final patch if there are more comments.